src/event/handlers/GhostPingMessageDeleteHandler.ts

```diff
@@ -0,0 +1,24 @@
+import { Constants, MessageEmbed, Message } from "discord.js";
+import { EMBED_COLOURS } from "../../config.json";
+import EventHandler from "../../abstracts/EventHandler";
+
+class GhostPingMessageDeleteHandler extends EventHandler {

```

Comment:
I think `GhostPingHandler` is more coherent with the stablished convention, besides being more concise. Anyways, good job Jacob :wink:

Reply:
I can fix it